kitsune/questions/views.py

```diff
@@ -468,7 +467,9 @@ def edit_details(request, question_id):
 def aaq_react(request):
     request.session['in-aaq'] = True
     to_json = JSONRenderer().render
-    products = ProductSerializer(Product.objects.filter(visible=True), many=True)
+    products = ProductSerializer(
+        Product.objects.filter(questions_locales__locale=request.LANGUAGE_CODE),
+        many=True)
```

Comment:
This mimics the filter in the `product_list` view, which controls which products are included on https://support.mozilla.org/questions. I imagine the reasoning there is because has-AAQ-enabled acts as a finer grained version of visible, and some locales may even want to open question forums for not-visible products.

But really it is just cargo culting. If I change this, I'd change the other one too.